src/vector_of_array.jl

```diff
@@ -638,8 +653,9 @@ end
 end
 
 Base.map(f, A::RecursiveArrayTools.AbstractVectorOfArray) = map(f, A.u)
+
 function Base.mapreduce(f, op, A::AbstractVectorOfArray)
```

Comment:
probably need to have an `init`

Reply:
Fixed in https://github.com/SciML/RecursiveArrayTools.jl/pull/325/commits/bdb58305372db57d38877e48157859e3137f7ccc